<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_drv">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Driver On/Off</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_refauto">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>auto reference</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_refY">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ref Y to 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_refX">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ref X to 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_refZ">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ref Z to 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
